Update Apps FAQ (#611)
btcpayserver/btcpayserver-doc: docs/FAQ/FAQ-Apps.md

* [What are the Apps in BTCPay?](#what-are-the-apps-in-btcpay)
* [Is there a limit on the number of Apps I can create?](#is-there-a-limit-on-the-number-of-apps-i-can-create)
* [Is there a Point of Sale feature in BTCPay?](#is-there-a-point-of-sale-feature-in-btcpay)
* [How can I use BTCPay in a physical store?](#how-can-i-use-btcpay-in-a-physical-store)
* [How to customize the appearance of Point of Sale App in BTCPay?](#how-to-customize-the-appearance-of-point-of-sale-app-in-btcpay)
* [What is a Payment Button?](#what-is-a-payment-button)
* [How to create a Pay Button with a custom amount?](#how-to-create-a-pay-button-with-a-custom-amount)
* [How to map a domain name to an app?](#how-to-map-a-domain-name-to-an-app)
* [How to fix empty spacing around payment button](#how-to-fix-empty-spacing-around-payment-button)
* [How to integrate WooCommerce Store in BTCPay Crowdfund app?](#how-to-integrate-woocommerce-store-into-a-btcpay-crowdfund-app)

## What are the Apps in BTCPay?

Apps are plugins (features) you can use to expand the use case of your BTCPay.

## Is there a limit on the number of Apps I can create?

> There's no limit. Apps are added on the store level. You need to have a store already created, to create an app which belongs to that store. You can create as many apps as you like for each store. 

Reviewer: There's a repetition of the word "create".
```suggestion
Apps are added on the store level. To create one, you need to have a store already setup. There is no limit to the number of apps that can be assigned to a store.
```

## Is there a Point of Sale feature in BTCPay?

Yes. Please read our [guide on creating the POS app](../WhatsNext.md#creating-the-pay-button).

## How can I use BTCPay in a physical store?

> You can use our Point of Sale (PoS) app. When you create a PoS app within BTCPay Server it will be publicly accessible via URL where checkout buttons for products you’ve created for your PoS will be displayed. For having a physical PoS, the easiest solution (currently) is to create a PoS App in BTCPay and display it on any web device such as a phone, tablet or pc.

Reviewer: ```suggestion
You can use our Point of Sale (PoS) app. When you create a PoS app within BTCPay Server, it will be publicly accessible via a URL where checkout buttons for products you’ve created for your PoS will be displayed.
To have a physical PoS, the easiest solution (currently) is to create a PoS App in BTCPay and display it on any web device such as a phone, tablet or pc.
```

Please follow our detailed guide on how to use our [PoS App on a mobile device](https://blog.btcpayserver.org/bitcoin-pos/). Also note that Section 2.3 Connecting a Wallet is covered much more in-depth here in the [wallet section](../WalletSetup.md).

## How to customize the appearance of Point of Sale App in BTCPay

It is very easy to customize the look of the Point of Sale app. [Follow this guide](../Theme.md) to learn how to change the theme.

## What is a Payment Button?

The Payment Button is a simple and customizable HTML button you can create and embed into your website. To create a payment button, [follow this guide](../WhatsNext.md#creating-the-point-of-sale-app). 

## How to create a Pay Button with a custom amount?

> BTCPay Pay Button which can be found in Store Settings > Pay Button, currently does not support custom amounts.

Reviewer: ```suggestion
The BTCPay Server Pay Button which can be found in Store Settings > Pay Button, currently does not support custom amounts.
```

However, you can use a work-around:

* [Create Point of sale app](../WhatsNext.md#creating-the-point-of-sale-app)
* Enable `user can input a custom amount` field
* Remove all the products from the automatically generated template.
* Save settings.
* Click on the `Embed payment button linking to PoS item` at the bottom of the page and copy the expanded code. Paste it into html page of your website.
* Remove the extra fields you do not need, especially `<input name="price" type="hidden" value="10" />` so that button redirects to the point of sale.

![Custom Amount Pay Button](../img/BTCPayPayButtonDynamic2.png)
![Custom Amount Pay Button](../img/BTCPayPayButtonDynamic.png)

## How to map a domain name to an app?

> BTCPay Apps can have a domain name that's different from the servers domain. Let's assume you have BTCPay server at mybtcpayserver.com and want to display your PoS app on mybtcpaypos.com instead mybtcpayserver.com/apps/pos/abc123

Reviewer: ```suggestion
BTCPay Server Apps can have a domain name that's different from the servers domain. Let's assume you have BTCPay Server at mybtcpayserver.com and want to display your PoS app on mybtcpaypos.com instead mybtcpayserver.com/apps/pos/abc123
```

If for any reason, you want an app to be on the same domain as your BTCPay Server homepage, you can select to display it on the root. In that case, no DNS configuration is needed, since your domain is already pointing properly. Using an app on the root domain of BTCPay Server means you'll have to access the login and other pages manually. The easiest way is to append a page route such as `/apps` or `/stores` to your root domain. (Ex: `mybtcpayserver.com/apps`). This will make navigation to your root displayed app easier, but navigation to other pages (such as Login) more challenging for users. 

## How to fix empty spacing around Payment Button?

This usually happens in Wordpress. The Wordpress text editor can cause conflicts with the pay button code by adding `<br>`, which is "line break" in HTML, between the hidden lines of the form, thus adding invisible empty lines.

You can get rid of that with a simple Wordpress plugin, [Don't muck my markup](https://wordpress.org/plugins/dont-muck-my-markup/). Install, activate it, and then you should see this box on the right of the edit page of your posts :

![Dont-muck-markup](../img/Dont-muck-markup.png)

> Just chose where you want to insert the pay button in, and the problem should be solved.

Reviewer: ```suggestion
Simply choose where you want to insert the pay button, and the problem should be solved.
```
